### ISSUE TITLE
In OS X, when zooming in, line number doesn't show properly

### ISSUE THREAD

**Issue by albusshin** [syl20bnr/spacemacs#4351]
I'm using emacs version 24.5.1 and the latest Spacemacs (commit  44d4525), v0.104.8

When I do `scale-font` operation on emacs and zoom in the text, the line number with the corresponding buffer doesn't show properly, since the line-number displaying region is not wide enough to show all the digits of a line number.

I don't know if this problem exists on other platforms. I'm using Mac OS X 10.11 (15A284)
![screen shot 2016-01-02 at 7 58 14 pm](https://cloud.githubusercontent.com/assets/1814327/12076827/32304390-b18b-11e5-942a-01f73b828601.png)
![screen shot 2016-01-02 at 7 58 49 pm](https://cloud.githubusercontent.com/assets/1814327/12076830/710d2eca-b18b-11e5-8cba-b2694f85f814.png)

**Comment by benquebec** [comment]
I get the same behaviour.
#### System Info
- OS: darwin
- Emacs: 24.5.6
- Spacemacs: 0.105.0
- Spacemacs branch: develop (rev. 8105e19)
- Distribution: spacemacs
- Layers:
```elisp
((colors :variables colors-enable-rainbow-identifiers t)
 emacs-lisp spell-checking syntax-checking osx)
```

**Comment by albusshin** [comment]
Just tried again on Ubuntu 14.04.3, this issue is reproducible.
Emacs version is 24.3.1

**Comment by d12frosted** [comment]
I confirm. The function you have to use in order to reproduce is `spacemacs/scale-up-font`.

**Comment by tonylotts** [comment]
I think this might be a bug with emacs itself.

**Comment by agzam** [comment]
this problem can be solved by using `zoom-frm-transient-state` `SPC z f`. Yet this thing doesn't properly work in fullscreen mode. You have to disable and re-enable fullscreen mode in order to use it. How to fix that I have no idea.

**Comment by kommen** [comment]
This doesn't reproduce any more for me: Emacs 26.1rc1, spacemacs current develop branch, macOS 10.13.4

**Comment by sdwolfz** [comment]
Oh, I remember this one, should be fixed in develop by #9209 ;).
